app/views/warehouse_reports/project/data_qualities/show.haml

```diff
@@ -1,48 +1,60 @@
-- content_for :title, _("Warehouse Reports")
-%h1= content_for :title
+- content_for :title, _('Project Data Quality Report')
+.d-flex
+  %h1= content_for :title
+  .ml-auto
+    = link_to history_warehouse_reports_project_data_quality_path, class: 'btn btn-secondary' do
+      %i.icon-settings_backup_restore
+      View Previous Reports
 
 = render 'breadcrumbs'
```

Comment:
Not sure there is anything to be done, but with the multi-step process, I clicked the breadcrumb to go back to step 1, and was surprised when I ended up back on the reports page...